pkg/health/health_aws.go

```diff
@@ -29,6 +30,32 @@ func GetAWSResourceHealth(resourceType, status string) (health HealthStatus) {
 }
 
 var awsResourceHealthmap = map[string]map[string]HealthStatus{
+	AWSResourceTypeCloudformationStack: {
```

Comment:
I don't think we need to map the statuses to fewer const's - we can just take the AWS status and titlecase it.